sdk/appconfiguration/Azure.Data.AppConfiguration/README.md

```diff
@@ -151,6 +158,64 @@ var client = new ConfigurationClient(connectionString);
 client.DeleteConfigurationSetting("some_key");
 ```
 
+### Create a Snapshot
+
+To create a snapshot, you need to instantiate the `ConfigurationSettingsSnapshot` class and specify filters to determine which configuration settings should be included. The creation process is a Long-Running Operation (LRO) and can be achieved by calling the `CreateSnapshot` method.
+
+```C# Snippet:AzConfigSample11_CreateSnapshot_AutomaticPolling
+List<SnapshotSettingFilter> snapshotFilter = new(new SnapshotSettingFilter[] { new SnapshotSettingFilter("some_key") });
```

Comment:
Either approach to initializing is fine, but I think we should be consistent within the samples.  It seems a bit odd to see both `var x = new SomeType` and `SomeType x = new();` in the same context.

Do you think it's reasonable for us to pick one style and use that across AppConfig, or is there a pattern that I'm not seeing?

Reply:
Thank you for bringing it to my attention! I've updated the sample to use var.